agimus_controller/main/main_hpp_panda_mpc_buffer.py

```diff
@@ -10,22 +9,23 @@
 from agimus_controller.trajectory_point import TrajectoryPoint, PointAttribute
 
 if __name__ == "__main__":
-    nq = 7
-    nv = 7
     pandawrapper = PandaWrapper(auto_col=True)
     rmodel, cmodel, vmodel = pandawrapper.create_robot()
     ee_frame_name = pandawrapper.get_ee_frame_name()
     hpp_interface = HppInterface()
-    ps = hpp_interface.get_panda_planner()
-    q_init = [pi / 6, -pi / 2, pi / 2, 0, 0, 0, -0.2, 0, 0.02, 0, 0, 0, 1]
-    whole_x_plan, whole_a_plan, whole_traj_T = hpp_interface.get_hpp_plan(
+    q_init, q_goal = hpp_interface.get_panda_q_init_q_goal()
+    hpp_interface.set_panda_planning(q_init, q_goal)
+    ps, viewer = hpp_interface.get_problem_solver_and_viewer()
+    whole_x_plan, whole_a_plan, whole_traj_T = hpp_interface.get_hpp_x_a_planning(
         1e-2, 7, ps.client.problem.getPath(ps.numberPaths() - 1)
```

Comment:
This arg should be removed and this value used inside the `get_hpp_x_a_planning` function

Reply:
the problem is that we don't retrieve the hpp path the same way for the panda robot and the ur3, for the panda we have :
`ps.client.problem.getPath()`
for the ur3 we have :
`ps.client.basic.problem.getPath()`